### PR TITLE
[Dashboard] Remove more Flex

### PR DESCRIPTION
**Pull request by kien-ngo**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR focuses on refactoring the layout components in various files from using `Flex` to `div` elements with Tailwind CSS classes for styling. This change aims to enhance consistency and potentially improve performance by simplifying the component structure.

### Detailed summary
- Replaced `Flex` components with `div` elements using Tailwind CSS classes.
- Adjusted layout properties to maintain the same visual design.
- Updated various components including `GraphicContainer`, `Timer`, `AddKeypairButton`, and others to ensure consistent styling.
- Ensured that all functional aspects remain intact after changes.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4784?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4784** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4784?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4765** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4765?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4750** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4750?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4749** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4749?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#4748** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4748?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kien-ngo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by MananTank — COMMENTED** [review]
nit: no need to specify `flex-row` everywhere. It's row by default

File: apps/dashboard/src/core-ui/batch-upload/upload-step.tsx [hunk 1]
```diff
@@ -33,22 +33,10 @@ export const UploadStep: React.FC<UploadStepProps> = ({
...
-              bg={hasFailed ? "red.200" : "inputBg"}
-              _hover={{
-                bg: hasFailed ? "red.200" : "inputBgHover",
-                borderColor: "blue.500",
-              }}
-              borderColor="inputBorder"
-              borderWidth="1px"
-              h="100%"
-=======
->>>>>>> 4a84e9e04 (temporarily revert Center eslint)
```
Comment (MananTank) [inline apps/dashboard/src/core-ui/batch-upload/upload-step.tsx:51]: This needs to be removed in the stack below - not in this PR

**Review by MananTank — CHANGES_REQUESTED** [review]
Move the merge conflict removal from this PR to PR in stack below this one

File: apps/dashboard/src/core-ui/batch-upload/upload-step.tsx (hunk 1 above)
Comment (kien-ngo) [reply apps/dashboard/src/core-ui/batch-upload/upload-step.tsx]: should be resolved. thanks

**Review by MananTank — APPROVED** [review]

**Pull request merged**